integrations/instructor_embedders/pydoc/config.yml

```diff
@@ -17,9 +18,9 @@ processors:
   - type: crossref
 renderer:
   type: haystack_pydoc_tools.renderers.ReadmePreviewRenderer
-  excerpt: Embedders integration for Haystack
+  excerpt: Instructor integration for Haystack
```

Comment:
Probably, it's better to specify Instructor embedders.
There is a different project called just "Instructor."